lib/osc-server.js

```diff
@@ -5,86 +5,94 @@ const osc = require('osc-min');
 
 export default class OscServer {
 
-    port = null;
-    ip = null;
-    sock = null;
-    consoleView = null;
-    subscribers: Map<string, Function> = new Map();
-    transformers: Map<string, Function> = new Map();
-
-    constructor(consoleView, ip, port) {
-        this.ip = ip;
-        this.port = port;
-        this.consoleView = consoleView;
-        this.sock = dgram.createSocket('udp4');
-    }
-
-    start(): Promise<void> {
-        let promise = new Promise((resolve, reject) => {
-            this.sock.on('listening', () => {
-                resolve()
-            });
-
-            this.sock.on('error', (err) => {
-                reject(err);
-            });
-        });
-
-        this.sock.bind(this.port, this.ip);
-
-        this.sock.on('message', (message) => {
-            let oscMessage = osc.fromBuffer(message);
-
-            if (oscMessage.oscType === "bundle") {
-              oscMessage.elements.forEach(element => {
-                this.handleOscMessage(element.address, element.args);
-              }
-              );
-            } else {
-              this.handleOscMessage(oscMessage.address, oscMessage.args);
-            }
+  port = null;
+  ip = null;
+  sock = null;
+  consoleView = null;
+  subscribers: Map<string, Function> = new Map();
+  transformers: Map<string, Function> = new Map();
+
+  constructor(consoleView, ip, port) {
+    this.ip = ip;
+    this.port = port;
+    this.consoleView = consoleView;
+    this.sock = dgram.createSocket('udp4');
+  }
+
+  start(): Promise<void> {
+    let promise = new Promise((resolve, reject) => {
+      this.sock.on('listening', () => {
+        resolve()
+      });
+
+      this.sock.on('error', (err) => {
+        reject(err);
+      });
+    });
+
+    this.sock.bind(this.port, this.ip);
+
+    this.sock.on('message', (message) => {
+      let oscMessage = osc.fromBuffer(message);
+
+      if (oscMessage.oscType === "bundle") {
+        oscMessage.elements.forEach(element => {
+          element.time = OscServer.fromNTPTime(oscMessage.timetag);
```

Comment:
better to pass the `timetag` to the subscriber and let them to convert it, if needed